### PR TITLE
Wcbranch

### PR DESCRIPTION
**Pull request by archerv-windcave**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by archerv-windcave** [comment]
Farmlands icon request to be added. The logo satisfied all the requirements mentioned in the guidelines.

**Comment by larouxn** [comment]
👋 Hello @archerv-windcave, it looks like there are some failing tests. You can see them under the [Ruby 3 CI](https://github.com/activemerchant/payment_icons/pull/451/checks?check_run_id=2680956710).

File: db/payment_icons.yml [hunk 1]
```diff
@@ -959,3 +959,6 @@
   label: UBP
   group: bank_transfers
   
+  name: farmlands_card
```
Comment (larouxn) [inline db/payment_icons.yml:962]: Need to add a `-` above this line like other icons.

**Review by larouxn — CHANGES_REQUESTED** [review]

**Comment by larouxn** [comment]
Hmm yeah the UBP test failure is rather strange. We'll definitely want to drop the change (https://github.com/activemerchant/payment_icons/pull/451/commits/1dc4d04c991a2f06d05645d143417d75b37f47ed) from this PR and potentially fix it upstream then rebase here.

Strangely I haven't been able to reproduce the "UBP should be Ubp" error yet.

**Comment by archerv-windcave** [comment]
I receive another error and I think the name of the card I added does not match with the file and I just change that already. The UBP came back and say expected UBP so I decided to change it back from what it was before. It could also be because of my new card because it sets just above the new added payment logo (I hope).
--
Archer Vallesfin
Software Developer

[Windcave]

T: +6493094693<tel:+6493094693>
D: +6499756273<tel:+6499756273>
E: ***@***.***
W: www.windcave.com
________________________________
WARNING - This email and any attachments may be confidential. If received in error, please delete and inform us by return email. Because emails and attachments may be interfered with, may contain computer viruses or other defects and may not be successfully replicated on other systems, you must be cautious. Windcave cannot guarantee that what you receive is what we sent. If you have any doubts about the authenticity of an email by Windcave, please contact us immediately. It is also important to check for viruses and defects before opening or using attachments. Windcave liability is limited to re-supplying any affected attachments. In addition, no employee of Windcave is authorized to agree upon binding contractual terms such as price via an email message unless such Windcave employee is a senior executive or unless such consent has been approved in writing by a Windcave senior executive. Any representation that violates our policy regarding email consent to contractual terms is null and void.

**Comment by larouxn** [comment]
```
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:78]:
{:message=>"The 'farmlands' SVG file does not have the appropriate <title> value"}.
Expected: "Farmlands Card"
  Actual: "Farmlands"
```

Hopefully we won't have to worry about UBP at all and can just focus on getting Farmlands Card ready.

**Comment by archerv-windcave** [comment]
Just make another pull request to fix that. I realize now that the first issue is somewhat related to the yml entry and not from svg itself. I hope it will pass now. Thanks.

--
Archer Vallesfin
Software Developer

[Windcave]

T: +6493094693<tel:+6493094693>
D: +6499756273<tel:+6499756273>
E: ***@***.***
W: www.windcave.com
________________________________
WARNING - This email and any attachments may be confidential. If received in error, please delete and inform us by return email. Because emails and attachments may be interfered with, may contain computer viruses or other defects and may not be successfully replicated on other systems, you must be cautious. Windcave cannot guarantee that what you receive is what we sent. If you have any doubts about the authenticity of an email by Windcave, please contact us immediately. It is also important to check for viruses and defects before opening or using attachments. Windcave liability is limited to re-supplying any affected attachments. In addition, no employee of Windcave is authorized to agree upon binding contractual terms such as price via an email message unless such Windcave employee is a senior executive or unless such consent has been approved in writing by a Windcave senior executive. Any representation that violates our policy regarding email consent to contractual terms is null and void.

**Comment by larouxn** [comment]
Yeah, I'll handle UBP if we need to do something. Seems to be alright actually.

Looks like CI passed now! 🙌 
Screen Shot 2021-05-27 at 12 18 51

**Review by larouxn — DISMISSED** [review]
Seems good to me. 🎩 
Screen Shot 2021-05-27 at 18 10 42

File: app/assets/images/payment_icons/farmlands.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...5-4.54-1.81-1.19-.32-2.69-.66-4.53-1.01-1.83-.4-3.44-.78-4.82-1.25-3.96-1.29-6.92-3.21-8.9-5.75-1.98-2.54-2.96-5.86-2.96-9.97a15 15 0 011.54-6.69c1.04-2.11 2.57-3.97 4.59-5.6 2.01-1.61 4.48-2.88 7.45-3.84 2.94-.94 6.28-1.4 10-1.4 3.54 0 6.81.37 9.77 1.09 2.97.76 5.48 1.66 7.51 2.7v13.51h-1.25c-.55-.44-1.39-1.02-2.55-1.74-1.17-.73-2.3-1.38-3.42-1.89-1.31-.62-2.74-1.14-4.38-1.53-1.65-.42-3.3-.63-4.97-.63-2.28 0-4.22.39-5.76 1.16-1.57.81-2.36 1.85-2.36 3.14 0 1.19.4 2.15 1.18 2.85.78.68 2.47 1.35 5.06 2.03 1.34.33 2.9.67 4.62 1.01 1.77.34 3.44.78 5.05 1.35 3.61 1.22 6.35 3.03 8.21 5.42 1.89 2.36 2.8 5.52 2.8 9.46 0 2.49-.55 4.87-1.67 7.1-1.1 2.26-2.7 4.15-4.68 5.71-2.19 1.65-4.74 2.96-7.65 3.86-2.91.9-6.44 1.37-10.52 1.37" id="pi-path239" fill="#00553d"/><path d="M188.05 243.46s84.44-17.16 164.72-17.16c118.1 0 140.45 11.56 225.82 18.33 36.62 3.11 67.9 4.49 69.75 4.67 3.56.31 9.34.39 9.34.39v-32.61H188.05v26.38" fill="#f8971d" id="pi-path241"/><path d="M188.05 251.98v18.98h469.63v-18.15s-59.21-2.8-92.88-5.57c-61.05-5.05-96.81-13.53-148.96-14.7-26.36-.57-49.79-2.18-114.33 2.91-12.03.96-56.89 6.25-77.95 10.36-7.71 1.54-35.51 6.17-35.51 6.17" id="pi-path243" fill="#00553d"/></g></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/farmlands.svg:1]: Do we need those id attributes?

- `id="pi-g258"` 
- `id="pi-g245"`
- `id="pi-path239"`
- `id="pi-path241"`
- `id="pi-path243"`
- `id="pi-path5444"`
- `id="pi-path5442"`
- `id="pi-svg5448"`

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

**Comment by archerv-windcave** [comment]
Hi,

I don't need those attributes. It's alright if removed.

--
Archer Vallesfin
Software Developer

[Windcave]

T: +6493094693<tel:+6493094693>
D: +6499756273<tel:+6499756273>
E: ***@***.***
W: www.windcave.com
________________________________
WARNING - This email and any attachments may be confidential. If received in error, please delete and inform us by return email. Because emails and attachments may be interfered with, may contain computer viruses or other defects and may not be successfully replicated on other systems, you must be cautious. Windcave cannot guarantee that what you receive is what we sent. If you have any doubts about the authenticity of an email by Windcave, please contact us immediately. It is also important to check for viruses and defects before opening or using attachments. Windcave liability is limited to re-supplying any affected attachments. In addition, no employee of Windcave is authorized to agree upon binding contractual terms such as price via an email message unless such Windcave employee is a senior executive or unless such consent has been approved in writing by a Windcave senior executive. Any representation that violates our policy regarding email consent to contractual terms is null and void.

**Comment by tauthomas01** [comment]
Yes, please remove those attributes.

**Comment by archerv-windcave** [comment]
Can you please check my changes Thomas. I already removed those unnecessary id's and also the conflict. Thank you.

**Comment by larouxn** [comment]
Looks like there's a test failure related to the `title` element.
```
Failure:
PaymentIconTest#test_Every_payment_SVG_meets_accessibility_requirements [/home/runner/work/payment_icons/payment_icons/test/unit/payment_icon_test.rb:78]:
{:message=>"The 'farmland' SVG file does not have the appropriate <title> value"}.
Expected: "Farmland"
  Actual: "Farmlands Card"
```
https://github.com/activemerchant/payment_icons/pull/451/checks?check_run_id=3409558962

**Comment by larouxn** [comment]
@archerv-windcave, are you alright with the margin spacing difference compared to other icons? If so, we can merge this.

**Comment by archerv-windcave** [comment]
Hi Nicholas,

I'm okay with the spacing, please merge the icon for us please.

Thank you for your awesome support.

Regards,
--
Archer Vallesfin
Software Developer

[Windcave]

T: +6493094693<tel:+6493094693>
D: +6499756273<tel:+6499756273>
E: ***@***.***
W: www.windcave.com
________________________________
WARNING - This email and any attachments may be confidential. If received in error, please delete and inform us by return email. Because emails and attachments may be interfered with, may contain computer viruses or other defects and may not be successfully replicated on other systems, you must be cautious. Windcave cannot guarantee that what you receive is what we sent. If you have any doubts about the authenticity of an email by Windcave, please contact us immediately. It is also important to check for viruses and defects before opening or using attachments. Windcave liability is limited to re-supplying any affected attachments. In addition, no employee of Windcave is authorized to agree upon binding contractual terms such as price via an email message unless such Windcave employee is a senior executive or unless such consent has been approved in writing by a Windcave senior executive. Any representation that violates our policy regarding email consent to contractual terms is null and void.

**Comment by larouxn** [comment]
👋 Just a heads up that we are hoping to have this live across Shopify within the month. Currently waiting on some other icons so we can batch the update. Thank you. 🙇‍♂️

**Comment by larouxn** [comment]
👋  Ran into a few release pipeline issues. Hoping to have this live by end of next week, potentially sooner.

**Pull request merged**